### PR TITLE
Bootstrap/instagram theme

### PR DESCRIPTION
**Pull request by iamlawrenceonoja**
# What does this PR do? 
This PR adds a new feature to Screenly's playground which is a Bootstrap theme for Screenly’s playground apps. The HTML file can be edited to serve multiple apps in the playground.

# What is the relevant GitHub Issue?

# Checklist

- [ ] I've added tests for the relevant code
- [x] The code passes linting (for the relevant language)
- [x] I've tested the design in all [supported resolutions](https://github.com/Screenly/playground/blob/master/docs/resolutions.md)
- [x] I've attached relevant screenshots (to speed up code review)
- [x] The final implementation is pixel perfect to the Figma design

# Screenshots
The screenshot below is the web page in landscape resolution using the default image and an example caption.

image

The screenshot below is the web page in portrait resolution using the default image and an example caption.
image

### CODE REVIEW

**Comment by vpetersson** [comment]
Please remove the Instagram readme. Basically, the Bootstrap theme is supposed to be the foundation for all our apps in the Playground. The Instagram app is just to the first use case for it, but the concept is much wider. It's intended to be the ideal Bootstrap theme for digital signage use cases.

**Comment by vpetersson** [comment]
@iamlawrenceonoja Also, please remove the `instagram_theme` sub-folder. The idea is to have a generic digital signage them. Instagram is just one use case. 

Moreover, please untick ` I've added tests for the relevant code` because you haven't added any tests.

File: bootstrap/index.html [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+                background: #0D6EFD;
+                font-family: 'Inter';
+                position: relative;
+            }
+            .header {
+                color: #FFFFFF;
+                font-size: 2.4vh;
+                font-family: 'Inter';
+                font-style: normal;
+                border-bottom: 0.1px groove white;
```
Comment (iamplugged) [inline bootstrap/index.html:31]: px values cannot be decimals

File: bootstrap/index.html [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+                margin: 0; 
+                height: 100%; 
+                overflow: hidden;
+                background: #0D6EFD;
+                font-family: 'Inter';
+                position: relative;
+            }
+            .header {
+                color: #FFFFFF;
+                font-size: 2.4vh;
```
Comment (iamplugged) [inline bootstrap/index.html:28]: I would recommend using rem units for font-size

File: bootstrap/index.html [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+                font-family: 'Inter';
+                font-style: normal;
+                bottom: 0.1vw;
+                width: 100%;
+                color: white;
+                text-align: center;
+                background-image: url("images/screenly_blur.svg");
+                mix-blend-mode:normal;
+                height: 50%;
+
```
Comment (iamplugged) [inline bootstrap/index.html:46]: Unnecessary newline here

File: bootstrap/index.html [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+                font-style: normal;
+                bottom: 0.1vw;
+                width: 100%;
+                color: white;
+                text-align: center;
+                background-image: url("images/screenly_blur.svg");
+                mix-blend-mode:normal;
+                height: 50%;
+                
+              }
```
Comment (iamplugged) [inline bootstrap/index.html:47]: Looks like extra spacing before closing braces

File: bootstrap/index.html [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+                font-style: normal;
+                bottom: 0.1vw;
+                width: 100%;
+                color: white;
+                text-align: center;
+            }
+
+            .instagram-post {
+                border-radius: 1vw 1vw 0vw 0vw;
+              }
```
Comment (iamplugged) [inline bootstrap/index.html:61]: Extra spacing before closing brace

File: bootstrap/index.html [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+              
+            .caption {
+              background: linear-gradient(0deg, rgba(255, 255, 255, 0.99), rgba(255, 255, 255, 0.99)), #0D6EFD;
```
Comment (iamplugged) [inline bootstrap/index.html:64]: Follow the same indentation throughout. I can see a mix of 2 spaces and 4 spaces.

File: bootstrap/index.html [hunk 7]
```diff
@@ -0,0 +1,217 @@
...
+                }
+                .footer {
+                    background: none;
+                }
+
+            }
+            
+            @media screen and (min-width:2160px) and (orientation:portrait) { 
+                .header-text {
+                    margin-top:0.9vh;
```
Comment (iamplugged) [inline bootstrap/index.html:118]: Needs space after colon

File: bootstrap/index.html [hunk 8]
```diff
@@ -0,0 +1,217 @@
...
+                }
+            }
+
+
+        </style>
+    
+    </head>
+    
+    <body>
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js"></script>
```
Comment (iamplugged) [inline bootstrap/index.html:141]: Can we move the scripts to the <head> or at the end of the body if we don't need to load them early.

File: bootstrap/index.html [hunk 9]
```diff
@@ -0,0 +1,217 @@
...
+            //Assign the Instagram Access token you generated to a variable.
+            var token = '{token}';
+        </script>
+
+        <!--Create divs to contain the images and captions-->
+    <div class="container-fluid">
+        
+        <div class="row d-flex justify-content-center mt-2 pb-3 header">
+            <div class="col-4">
+                <img class="playground-logo" width="40%" height="auto" src="images/playground_logo.svg" alt="Screenly logo" style="float:left;">
```
Comment (iamplugged) [inline bootstrap/index.html:152]: If there's no particular reason for having the style inline, please move to styles section.

File: bootstrap/index.html [hunk 10]
```diff
@@ -0,0 +1,217 @@
...
+            </div>
+            <div class="col-4 pt-3 header-text">
+                <span>Instagram Feed</span>
+            </div>
+        </div>
+
+        <div class="row d-flex justify-content-center mt-5 pt-3">   
+            <div id="frame0" class="col-sm-4 frames">
+                <img id="image-zero" class="img-responsive instagram-post" src="images/default_card.svg" alt="Instagram Image">
+                <div class="col-12 caption" id="caption-zero"><b style="color:#0D6EFD;">@screenly</b> Text Placeholder</div>
```
Comment (iamplugged) [inline bootstrap/index.html:162]: Same here. Move inline styles to styles section

**Comment by iamlawrenceonoja** [comment]
https://github.com/Screenly/playground/pull/51

This is the PR to be reviewed please.
I closed this and created a new one.

File: bootstrap/index.html [hunk 11]
```diff
@@ -0,0 +1,217 @@
...
+
+        </div>
+    </div>
+    <div class="footer"></div>
+    <div class="footer-text">© 2013-2022 Screenly, Inc.</div>
+
+  
+    </body>
+
+    <script>
```
Comment (iamplugged) [inline bootstrap/index.html:178]: Get rid of this section if you don't plan to use it

**Pull request closed without merging**